docs/en-US/component/anchor.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Anchor
+lang: en-US
+---
+
+# Anchor
+
+Through the anchor point, you can quickly find the position of the information content on the current page.
+
+## Basic Usage
+
+The most basic usage
+
+:::demo
+
+anchor/basic
+
+:::
+
+## Horizontal Mode
+
+Horizontally aligned anchors
+
+:::tip
+
+Horizontal Mode does not support sublist slots
+
+:::
+
+:::demo
+
+anchor/horizontal
+
+:::
+
+## Affix Mode
+
+Use the affix component to fix the anchor point within the page.
+
+:::demo
+
+anchor/affix
+
+:::
```

Comment:
Move the affix mode to the last position, avoid shading.

Reply:
done